sunbeam-python/sunbeam/plugins/observability/etc/deploy-grafana-agent/variables.tf

```diff
@@ -0,0 +1,40 @@
+# Terraform manifest for deployment of Grafana Agent
+#
+# Copyright (c) 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "principal-application" {
+  description = "Name of the deployed principal application that integrates with grafana-agent"
+}
+
+variable "controller-model" {
+  description = "Name of the sunbeam controller model"
+  default = "controller"
+}
+
+variable "grafana-agent-channel" {
+  description = "Channel to use when deploying grafana agent machine charm"
+  default = "latest/candidate"
```

Comment:
Can you add a comment above that channel latest/stable not available for grafana-agent and so defaulted to latest/candidate